packages/spotlib/spotlib.2.5.3/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+authors: [ "jun.furuse@gmail.com" ]
```

Comment:
Thanks @Chris00.  I use `omake` therefore not a big fan of Oasis and am not really motivated to provide a good `_oasis`.  Anyway, let me check my build scripts to see it is easy to transfer some information of `opam` and `descr` to `_oasis`.